wire-ios-request-strategy/Sources/Payloads/Processing/ConversationEventPayloadProcessor.swift

```diff
@@ -25,6 +26,12 @@ final class ConversationEventPayloadProcessor {
         case eventStream
     }
 
+    private let localConversationRemovalUseCase: LocalConversationRemovalUseCaseProtocol
```

Comment:
we also drop the UseCase suffix for properties
```suggestion
    private let removeLocalConversation: LocalConversationRemovalUseCaseProtocol
```

Reply:
done